akka-http-core/src/main/scala/akka/http/scaladsl/model/headers/CacheDirective.scala

```diff
@@ -86,22 +86,25 @@ object CacheDirectives {
   case object `must-revalidate` extends SingletonValueRenderable with ResponseDirective
 
   // http://tools.ietf.org/html/rfc7234#section-5.2.2.2
-  final case class `no-cache`(fieldNames: immutable.Seq[String]) extends FieldNamesDirective with ResponseDirective
+  final case class `no-cache`(fieldNames: colImmutable.Seq[String]) extends FieldNamesDirective with ResponseDirective
 
   // http://tools.ietf.org/html/rfc7234#section-5.2.2.5
   case object `public` extends SingletonValueRenderable with ResponseDirective
 
+  // https://tools.ietf.org/wg/httpbis/draft-ietf-httpbis-immutable/
```

Comment:
Perhaps also make the comment an actual scaladoc, so people can read up on it (without having to get sources)